src/components/home/Ecosystem.js

```diff
@@ -44,10 +44,10 @@ const Ecosystem = () => (
   <section className="home__part home__ecosystem">
     <div className="container">
       <h1 className="ecosystem__title">
-        The API Platform <strong>framework</strong>
+        The API Platform <strong>Framework</strong>
       </h1>
       <h5>
-        API Platform is a set of tools to help you building API-first projects
+        API Platform is a set of tools build and consume web APIs
```

Comment:
API Platform is a set of tools to build and consume web APIs

Reply:
missing word `to`